Source/Core/Core/PowerPC/Jit64Common/Jit64AsmCommon.cpp

```diff
@@ -404,226 +404,208 @@ void CommonAsmRoutines::GenQuantizedSingleStores()
 	singleStoreQuantized[7] = storeSingleS16;
 }
 
-void CommonAsmRoutines::GenQuantizedLoads()
+const u8* CommonAsmRoutines::GenQuantizedLoadRuntime(bool w, EQuantizeType type)
 {
 	const void* start = GetCodePtr();
+	JitRegister::Register(start, GetCodePtr(), "JIT_QuantizedLoad_%i_%i", type, w);
```

Comment:
This will register a zero lenght. This call must be move to the end of this function.